crates/bevy_ui/src/ui_node.rs

```diff
@@ -238,15 +238,19 @@ pub enum ValArithmeticError {
 
 impl Val {
     /// A convenience function for simple evaluation of [`Val::Percent`] variant into a concrete [`Val::Px`] value.
```

Comment:
This line should probably be updated to reflect that it is not just the `Val::Percent` variant that is resolved.